kadasrouting/gui/navigationpanel.py

```diff
@@ -408,6 +536,21 @@ def startNavigation(self):
 
         self.setMessage(self.tr("Connecting to GPS..."))
         self.gpsConnection = getGpsConnection()
+        try:
+            if iface.activeLayer().name() == "Routes":
```

Comment:
Is it always "Routes"?. I think we can rename the layer.

Reply:
yes, it is...

Reply:
KADAS creates one layer called `Routes` (not possible to change) for both waypoints _and_ routes...